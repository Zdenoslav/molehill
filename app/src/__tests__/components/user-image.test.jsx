import React from 'react';
import {shallow} from 'enzyme';

import UserImage from 'app/components/user-image';

describe('User Image', () => {

  it('should render UserImage with size', () => {
    const wrapper = shallow(<UserImage imgSize={50} imgSrc="img-source" />);
    expect(wrapper).toMatchSnapshot();
  });

});
