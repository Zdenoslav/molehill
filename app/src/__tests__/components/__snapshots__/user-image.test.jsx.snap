// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Image should render UserImage with size 1`] = `
<img
  height={50}
  src="img-source"
  width={50}
/>
`;
